.claude/hooks/utils/llm/ollama_provider.py

```
@@ -0,0 +1,193 @@
+# dependencies = [
+#     "ollama",
+#     "python-dotenv",
+# ]
+# ///
+
+import os
+import sys
+from dotenv import load_dotenv
+
+
+def prompt_llm(prompt_text):
```

Ruff (0.12.2)

`prompt_llm` is too complex (16 > 10)

(C901)